runtime/runtime/src/actions.rs

```diff
@@ -472,10 +418,8 @@ pub(crate) fn check_actor_permissions(
                 }
                 .into());
             }
-            if actor_id != account_id
-                && check_rent(account_id, account, config, apply_state.epoch_length).is_ok()
-            {
-                return Err(ActionErrorKind::DeleteAccountHasRent {
+            if actor_id != account_id && check_storage_cost(account, config).is_ok() {
```

Comment:
You shouldn't be able to delete someone's account, because they shouldn't be able to reach such state.

Reply:
Fixed.